plugins/API/Renderer/Original.php

```diff
@@ -100,8 +100,20 @@ private function serializeIfNeeded($response)
                 });
             }
 
-            return serialize($response);
+            $response = serialize($response);
+            if ($this->hideIdSubDataTable) {
+                $response = $this->removeIdSubDataTables($response);
+            }
+            return $response;
         }
         return $response;
     }
+
+    public function removeIdSubDataTables($response)
+    {
+        $response = preg_replace('/"subtableId";i:[\d]+;/', '', $response);
```

Comment:
@katebutler to possibly avoid regexes would it be better to do something similar as in PHP renderer where we iterate over all rows and then call in this case `$row->removeSubtable()` like

```php
foreach ($table->getRows() as $row) {
 if ($this->hide...) {   $row->removeSubtable(); }
}
```

haven't tested but could work